sycl/include/CL/sycl/accessor.hpp

```diff
@@ -839,8 +839,17 @@ class __SYCL_SPECIAL_CLASS accessor :
     size_t Result = 0;
     // Unroll the following loop for both host and device code
     __SYCL_UNROLL(3)
-    for (int I = 0; I < Dims; ++I)
-      Result = Result * getMemoryRange()[I] + getOffset()[I] + Id[I];
+    for (int I = 0; I < Dims; ++I) {
+      Result = Result * getMemoryRange()[I] + Id[I];
+#if __cplusplus >= 201703L
```

Comment:
It looks like the property is silently ignored if DPC++ code is built in C++14 mode. Can we document this fact somehow?

Reply:
According to specification, that feature can't be used in C++14. Code just will not be compilled in C++14.